bin/test-documentation

```diff
@@ -11,7 +11,9 @@ EXTRA_TASK=""
 # Check if it is a scheduled build
 if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
     # check external links as well
-    EXTRA_TASK="markdownValidateExternalLinks"
+    # Disabled for 1.5.x. It seems to introduce more noise than value to check links for old
+    # documentation pages.
+    # EXTRA_TASK="markdownValidateExternalLinks"
 fi
```

Comment:
Meh, this appears to e invalid bash: `bin/test-documentation: line 17: syntax error near unexpected token `fi'`

https://travis-ci.com/github/lagom/lagom/jobs/308669757#L856

I wouldn't have guessed, TBH.

Reply:
Oops, I'll just remove all of it.